snapshot/spec/test_command_generator_spec.rb

```diff
@@ -17,7 +17,8 @@ def configure(options)
 
         it "uses the default parameters" do
```

Comment:
Since I don't have an Apple TV app myself, I'd love some tests for Apple TV so we don't break anything by mistake. Also including a sample project in the `examples` folder would be awesome. It can be completely empty with just one view.